### PR TITLE
Fix 'Failed check: Ellipsis' on Weblate

### PR DESCRIPTION
**Pull request by luixxiul**
Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

This PR fixes [`Failed check: Ellipsis` on Weblate](https://translate.element.io/translate/element-web/matrix-react-sdk/en/?q=check:ellipsis). Ideally there should be a script to lint and fix automatically.

type: task

## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [ ] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/i18n/strings/en_EN.json [hunk 1]
```diff
@@ -3426,7 +3426,6 @@
     "Mark as not suggested": "Mark as not suggested",
     "Mark as suggested": "Mark as suggested",
     "Failed to remove some rooms. Try again later": "Failed to remove some rooms. Try again later",
-    "Removing...": "Removing...",
```
Comment (luixxiul) [inline src/i18n/strings/en_EN.json:3429]: Removed as duplicate of line 2716

**Comment by luixxiul** [comment]
Found some other cases to be fixed.

**Comment by luixxiul** [comment]
There are still cases to be fixed, which require the code to be changed so that the plural form is automatically applied to the file for Weblate. Leaving it for a future fix.

**Review by robintown — APPROVED** [review]
Thank you! I'll need to ask the Web App Team whether it's appropriate to bypass the coverage requirement in this case.

**Comment by andybalaam** [comment]
Yep, all good to bypass coverage requirements for this type of change. Merging.

**Pull request merged**